.github/actions/pre-commit/action.yml

```
@@ -29,6 +37,18 @@ runs:
         ${{ inputs.pre-commit-all-files }} && EXTRA_ARGS='--all-files'
         EXTRA_ARGS="${EXTRA_ARGS} ${{ inputs.pre-commit-args }}"
         echo EXTRA_ARGS="${EXTRA_ARGS}" >> $GITHUB_ENV
+
     - uses: pre-commit/action@v3.0.0
+      id: pre-commit
+      continue-on-error: true
```

```suggestion
      continue-on-error: ${{ inputs.auto-commit == 'true' }}
```

Not sure if that actually works, but I'd rather avoid re-throwing if auto-commit is disabled anyway, wdyt?

---

looks cool indeed, let me try it.